README.adoc

```diff
@@ -0,0 +1,57 @@
+= MojoHaus TagList Maven Plugin
+
+This is the http://www.mojohaus.org/taglist-maven-plugin/[taglist-maven-plugin].
+
+image:https://img.shields.io/github/license/mojohaus/taglist-maven-plugin.svg?label=License["Apache License, Version 2.0, January 2004",link=http://www.apache.org/licenses/]
+image:https://img.shields.io/maven-central/v/org.codehaus.mojo/taglist-maven-plugin.svg?label=Maven%20Central[Maven Central,link=http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22org.codehaus.mojo%22%20AND%20a%3A%22taglist-maven-plugin%22]
+image:https://github.com/mojohaus/taglist-maven-plugin/workflows/GitHub%20CI/badge.svg?branch=master[Build Status,link=https://github.com/mojohaus/taglist-maven-plugin/actions/workflows/maven.yml?query=branch%3Amaster]
+
+== Contributing
+
+=== Creating Issues
+
+If you find a problem please create an
+https://github.com/mojohaus/taglist-maven-plugin/issues[issue in the ticket]
+and describe what is going wrong or what you expect to happen.
+If you have a full working example or a log file this is also helpful.
+You should of course describe only a single issue in a single ticket and not
+mixing up several different things into a single issue.
+
+=== Creating a Pull Request
+
+Before you create a pull request it is necessary to create an issue in
+the https://github.com/mojohaus/taglist-maven-plugin/issues[ticket system]
+and describe what the problem is or what kind of feature you would like
+to add. Afterwards you can create an appropriate pull request.
+
+It is required if you want to get a Pull request to be integrated into please
+squash your commits into a single commit which references the issue in the
+commit message which looks like this:
+
+----
+Fixed #Issue
+
+ o Description.
+
+----
+
+This makes it simpler to merge it and this will also close the
+appropriate issue automatically in one go. This make the life as
+maintainer a little bit easier.
+
+A pull request has to fulfill only a single ticket and should never
+create/add/fix several issues in one, cause otherwise the history is hard to
+read and to understand and makes the maintenance of the issues and pull request
+hard or to be honest impossible.
+
+== Releasing
+
+* Make sure `gpg-agent` is running.
+* Execute `./mvnw -B release:prepare release:perform`
+
+For publishing the site do the following:
+
+----
+cd target/checkout
+../mvnw verify site site:stage scm-publish:publish-scm
```

Comment:
Except the mvnw...

Reply:
True. Let's wait for the next Maven 3 mvnw release, WDYT?